docs/development/benchmark.njk

```diff
@@ -1,3 +1,7 @@
+---
+permalink: '/development/benchmark.html'
+---
+
```

Comment:
These `permalink` front matter properties are necessary to keep the exact same structure that the site had before.

- Without `permalink`: Generates as `_site/development/benchmark/index.html`
- With `permalink`: Generates as `_site/development/benchmark.html`